### ISSUE TITLE
npm WARN react-native-image-resizer@0.1.1 requires a peer of react@^15.4.0 but none was installed.

### ISSUE THREAD

**Issue by npomfret** [bamlab/react-native-image-resizer#80]
Can we remove [this](https://github.com/bamlab/react-native-image-resizer/blob/master/package.json#L21) dependency from the build?  I don't think it's valid.

**Comment by 4ian** [comment]
I'm not sure about it, looking at react-native-fs [package.json](https://github.com/itinance/react-native-fs/blob/master/package.json), they included react and react-native in the `devDependencies`. Do you know what the best practice is ?

**Comment by 4ian** [comment]
Same thing done here: https://github.com/wix/react-native-navigation/blob/master/package.json and here: https://github.com/airbnb/react-native-maps/blob/master/package.json so I guess it's the best way!

**Comment by dgurns** [comment]
I'm getting this error and can't import the module into my project at all. Getting the error "Unable to resolve path to module"

I'm on React Native 0.44.0 and React 16.0.0-alpha.6

**Comment by 4ian** [comment]
@dgurns  Can you post the full command + output leading to the error?

**Comment by leethree** [comment]
I think the warning is because `react-native` uses `react` 16, which doesn't satisfy `^15.4.0`. Including version `^16.0.0-alpha` to peer dependency should fix the issue. Something like this:

```json
"peerDependencies": {
  "react": "^15.4.0 || ^16.0.0-alpha",
}
```

**Comment by leethree** [comment]
this should already be fixed by #96

**Comment by PierreCapo** [comment]
Closed as staled issue with old version of React-Native or react-native-image-resizer.
